### DISCUSSION TITLE
Is there is a way to pass image metadata as training features?

### DISCUSSION THREAD

**Discussion by OmarAshkar** [Project-MONAI/MONAI#7673] [Q&A]
Hello,
I was wondering if there is an easy way in MONAI to pass non-image features (like patient age, weight ...etc) to training? I want to use a vanilla UNET for now, but probability will need to used arbitrary input NN later.

TIA,
Omar

**Comment by KumoLiu** [comment]
Hi @OmarAshkar,

There may not be a simple way to accomplish this; you must go and modify your network. However, you can enter metatensor as the network input and then get the meta information you want using `data.meta[key]`.


Hope it helps, thanks.
